doc/user/load_save.md

```diff
@@ -523,6 +523,77 @@ outImages.push_back("bandicoot-favicon-inv.jpeg");
 mlpack::data::Save(outImages, matrix, info);
 ```
 
+### Resize images
+
+It is possible to resize images in mlpack with the following function:
+
+- `ResizeImages(images, info, newWidth, newHeight)`
+   * `images` an armadillo matrix contains a set of images, each one is
+     represeted in one column representing the image as a flattened vector.
+
+   * `info` will be populated with information from the image in `image`.
+
+   * `newWidth` the new width of the resized image.
+
+   * `newHeight` the new height of the resized image.
```

Comment:
```suggestion
   * `images` is a [column-major matrix](matrices.md) containing a set of
      images; each image is represented as a flattened vector in one column.

   * `info` is a [`data::ImageInfo&`](#dataimageinfo) containing details about
     the images in `images`, and will be modified to contain the new size of the
     images.

   * `newWidth` and `newHeight` (of type `size_t`) are the desired new
     dimensions of the resized images.
```

I tried to make these read a little more like the other documentation; hopefully I did not mess up any intentions you had for the writing.  I also wanted to make sure to clarify the types.